### DISCUSSION TITLE
Spellcheck dictionaries location and implementation on Linux Mint

### DISCUSSION THREAD

**Discussion by snden** [vkbo/novelWriter#2074] [Q&A]
Hello, I'm trying to find out which spellcheck dictionaries NovelWriter uses on my system (Linux Mint). Where are these dictionaries located and how can I see the list of available dictionaries? The GUI only shows 'Spell check language - Available languages determined by your system' but I'd like to know more details about the actual implementation.

**Comment by vkbo** [comment]
It's covered in the docs: https://novelwriter.io/docs/more_customise.html#a-custom-dict

TLDR: On Linux and Mac, hunspell, aspell and myspell dictionaries from the system should show up. On Windows they must be installed manually, but a tool is provided to help with it.

**Reply by snden** [reply]
Thank you for your quick and detailed response! This helps a lot to understand how the spell checking system works in NovelWriter. Now I know that it uses Enchant and can work with hunspell dictionaries which I already have installed on my Linux Mint system.

**Reply by jartheb** [reply]
> https://novelwriter.io/docs/more_customise.html#a-custom-dict

404 on the link. Could we get an updated doc?

**Reply by vkbo** [reply]
It's right there in the side menu of the docs if you just click away from the 404 and look, but here it is: https://novelwriter.io/docs/more/dictionaries.html
